### ISSUE TITLE
Arithmetic overflow in timestamp calculation

### ISSUE THREAD

**Issue by Boddlnagg** [Boddlnagg/midir#8]
```
thread '<unnamed>' panicked at 'arithmetic operation overflowed', ../midir/src/alsa/mod.rs:748
```

I'm not sure what to do about this.

**Comment by Boddlnagg** [comment]
This is probably no longer relevant after #29.
